guide/english/machine-learning/pytorch-for-deep-learning/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Deep Learning using PyTorch
+---
+
+
+#### What is PyTorch?
+"PyTorch is open source deep learning platform primarily developed by Facebook's artificial-intelligence research group,that provides a seamless path from research prototyping to production deployment which has been adopted by many (if not most) of the researchers that we most respect."

```

Comment:
Can we rephrase this as:
"PyTorch is **an** open source deep learning platform **, which is** primarily developed by Facebook's artificial intelligence research group **. It** provides......."